Source/WebCore/animation/CSSPropertyAnimation.cpp

```diff
@@ -3496,6 +3496,7 @@ CSSPropertyAnimationWrapperMap::CSSPropertyAnimationWrapperMap()
         case CSSPropertyContainer:
         case CSSPropertyContainerName:
         case CSSPropertyContainerType:
+        case CSSPropertyContentVisibility:
```

Comment:
Can you file a bug to add animation support for this new property?

Reply:
I went with renaming the bug to be in line with https://bugs.webkit.org/show_bug.cgi?id=241184.